app/src/main/java/com/sopt/now/Home/HomeFragment.kt

```diff
@@ -0,0 +1,52 @@
+package com.sopt.now.Home
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.ConcatAdapter
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.sopt.now.Friend.FriendAdapter
+import com.sopt.now.User.UserAdapter
+import com.sopt.now.databinding.FragmentHomeBinding
+
+
+
+class HomeFragment: Fragment() {
+    private var _binding: FragmentHomeBinding? = null
+    private val binding: FragmentHomeBinding
+        get() = requireNotNull(_binding)
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentHomeBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+    private val viewModel by viewModels<HomeViewModel>()
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val friendAdapter = FriendAdapter()
+        val userAdapter = UserAdapter()
+
+        friendAdapter.setFriendList(viewModel.mockFriendList)
+        userAdapter.setUserList(viewModel.mockUserList)
+        val concatAdapter = ConcatAdapter(userAdapter, friendAdapter)
+
+        binding.rvFriends.run {
+            adapter = concatAdapter
+            layoutManager = LinearLayoutManager(requireContext())
+        }
+
+
+    }
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
+    }
```

Comment:
```suggestion
    override fun onDestroyView() {
        _binding = null
        super.onDestroyView()
    }
```

많은 오비분들이 리뷰 달고 다니실거같은데!! 이쪽이 메모리 누수에 안정적이라고 하네요 😄

Reply:
오 이거는 몰랐네요! 저도 수정해야겠어요!

Reply:
공식문서에서는 상단의 코드로 나오기는 합니다.

수명주기 메소드를 호출한 다음에 변수를 null 처리 하는 것이 저는 좀더 낫다고 생각해요!